en/concept/architecture.md

```diff
@@ -75,7 +75,7 @@ Its goal is to adjust the value of the process variable such that it matches the
 The output is a correction to eventually reach that setpoint.
 For example the position controller takes position setpoints as inputs, the process variable is the currently estimated position, and the output is an attitude and thrust setpoint that move the vehicle towards the desired position.
 
-A **mixer** takes force commands (such as "turn right") and translates them into individual motor commands, while ensuring that some limits are not exceeded.
+A **mixer** takes force commands (such as "turn right") and translates them into individual motor commands, while ensuring that some limits are not exceeded. 
```

Comment:
That can stay - the concept is the same (we're generally using mixer and allocation interchangeably)